prow/jobs/images/Dockerfile.test

```diff
@@ -75,7 +75,8 @@ RUN curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.10.0/kind-linux-amd64 && \
 
 RUN curl -fsSL https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip \
     && unzip awscliv2.zip \
-    && aws/install
+    && aws/install \
+    && export AWS_PAGER=""
```

Comment:
TIL.

Reply:
Yeah I was wondering why AWS CLI v2 inside the container scripts was not returning anything on stdout. Turns out it defaults to `less`, which is a crazy decision that is buried in the migration docs.